packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/telecom-telephony-billing-account.html

```diff
@@ -23,44 +23,53 @@
                 <oui-header-tabs>
                     <oui-header-tabs-item
                         data-text="{{ 'telephony_group_tab_dashboard' | translate }}"
-                        data-state="telecom.telephony.billingAccount">
+                        data-href="{{:: BillingAccountCtrl.billingAccountLink }}"
+                        data-active = "BillingAccountCtrl.currentActiveLink() === BillingAccountCtrl.billingAccountLink">
```

Comment:
:warning: Caution there are some extra spaces.

```suggestion
                        data-active="BillingAccountCtrl.currentActiveLink() === BillingAccountCtrl.billingAccountLink">
```

Reply:
Where is the Prettier poc ? @antleblanc  😅

Reply:
This should have been detected by a linter 🤔

Reply:
It's a valid syntax!
But to keep consistency please prefer using the same coding style.